src/core/json/json_object.cc

```diff
@@ -25,7 +25,10 @@ optional<JsonType> JsonFromString(string_view input, PMR_NS::memory_resource* mr
   parser.finish_parse(decoder, ec);
 
   if (decoder.is_valid()) {
-    return decoder.get_result();
+    auto res = decoder.get_result();
+    // This should not trigger once pmr is fixed in jsoncons
+    DCHECK(res.get_allocator().resource() == mr);
```

Comment:
we are blocked on this until it's addressed by jsoncons

Reply:
so add an issue in the comment and comment out the DCHECK I guess?

Reply:
it will cause a memory leak. We will loose this `mr`. Yes maybe we can release it here but then the tracking will be wrong. :/